build/dt-report-generator-bundle.js

```diff
@@ -7,13 +7,35 @@
 
 const browserify = require('browserify');
 const fs = require('fs');
+const rimraf = require('rimraf');
 
-const distDir = __dirname + '/../dist';
-const outFile = `${distDir}/report-generator.js`;
+const distDir = __dirname + '/../dist/dt-resources';
```

Comment:
nit: I know it was pre-existing but let's `path.join` this sucker :)

Reply:
meant to do this in the last one?

Reply:
ya, done